src/index.html

```diff
@@ -15,8 +15,111 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header
+      id="top"
+      class="header"
+    >
+      <div class="headerImgContainer">
```

Comment:
Tutaj tego diva mogłoby nie byc bo on nadaje tylko height: 40px; co tak na prawde mogłbys dac w klasie headerImg.

Reply:
Ok, to zrobię tak że dodam klasę .headerImg na poziomie elementu "a" bo na poziomie "img" height nie ma efektu i wyrzucam div.